.github/workflows/pr.yml

```diff
@@ -0,0 +1,1687 @@
+name: PR Build
+on:
+  workflow_dispatch:  # Add this line to enable manual triggering
+  pull_request:
+    paths-ignore:
+    - '**.md'
+env:
+  REALM_DISABLE_ANALYTICS: true
+  # CMAKE_C_COMPILER: /usr/local/opt/ccache/libexec/clang
+  # CMAKE_CXX_COMPILER: /usr/local/opt/ccache/libexec/clang++
+  CMAKE_C_COMPILER: /usr/local/bin/ccache-clang
+  CMAKE_CXX_COMPILER: /usr/local/bin/ccache-clang++
+  RELEASE_BRANCHES: "[ 'master', 'releases', 'feature/github-actions' ]"
+
```

Comment:
You might want to add something like

```
concurrency:
  group: ${{ github.workflow }}-${{ github.ref }}
  cancel-in-progress: true
```

This will stop the current build if a new commit is pushed

Reply:
Great tip. I wasn't aware that was possible 👍